calculator.rb

```diff
@@ -0,0 +1,67 @@
+print "Welcome to my calculator program! Please choose an operation 
+you would like to calculate\n"
+
+#Array with operations available
+#At some point figure out how to print directly from array so as to not repeat myself with puts
+# Change this into an array so a user could input other mathematical operations
+operations = ["add", "+", "subtract", "-", "multiply", "*", "divide", "/"]
+
+puts "1. add(+) "
+puts "2. substract(-)"
+puts "3. mulitply(*)"
+puts "4. divide(/)"
+
+# Ask user to describe what operation performed. YOu will have to build in a feature that 
+#creates a way to check user input for correct format.
+
+print "Please choose an operator (name or symbol)"
+user_choice = gets.chomp
+choice = false 
+while choice == false
+    if operations.include?(user_choice)
+        choice = true
+        puts "success"
+    else
+       puts "Please enter a correct operation"
+       user_choice = gets.chomp
+    end
+end
+ 
+# Ask user for their numbers, check if integer
+
+print "What is your first number: "
+begin
+  num_one = gets.chomp
+  num_one= Integer(num_one)
+rescue ArgumentError
+  print "Please enter a number:"
+  retry
+end
+print num_one
```

Comment:
Because you `print num_one` here, it ends up looking awkward in the terminal, because `num_one` gets displayed even though I (the user) just typed it! Maybe we can get rid of this line?